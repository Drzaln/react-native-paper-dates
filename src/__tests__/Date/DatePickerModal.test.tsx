import { render } from '@testing-library/react-native'
import { SafeAreaProvider } from 'react-native-safe-area-context'
import DatePickerModal from '../../Date/DatePickerModal'

it('renders DatePickerModal', () => {
  const { toJSON } = render(
    <SafeAreaProvider>
      <DatePickerModal
        locale="en"
        mode="single"
        visible={true}
        onDismiss={() => null}
        onConfirm={() => null}
      />
    </SafeAreaProvider>
  )
  expect(toJSON()).toMatchSnapshot()
})
