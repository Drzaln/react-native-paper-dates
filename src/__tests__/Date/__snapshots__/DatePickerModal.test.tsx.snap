// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DatePickerModal 1`] = `null`;
